DependencyInjection/DoctrinePHPCRExtension.php

```diff
@@ -20,6 +20,7 @@
 
 namespace Doctrine\Bundle\PHPCRBundle\DependencyInjection;
 
+use PHPCR\ConfigurationException;
```

Comment:
sorry, this is the wrong exception. it should be Symfony\Component\Config\Definition\Exception\InvalidConfigurationException